Formula/d/dud.rb

```diff
@@ -0,0 +1,18 @@
+class Dud < Formula
+  desc "CLI tool for versioning data"
+  homepage "https://kevin-hanselman.github.io/dud/"
+  url "https://github.com/kevin-hanselman/dud/archive/refs/tags/v0.4.5.tar.gz"
+  sha256 "57f63d260c8a0a0f925bb67e3634f1211c69b07a7405215b53c38d8563119425"
+  license "BSD-3-Clause"
```

Comment:
add head build support

```suggestion
  license "BSD-3-Clause"
  head "https://github.com/kevin-hanselman/dud.git", branch: "main"
```